### ISSUE TITLE
[FR] Allowing secondarieS Y-axes

### ISSUE THREAD

**Issue by Nickduino** [kalkih/mini-graph-card#1170]
(Yes, I checked, axes is the plural of axis)

Instead of allowing one secondary Y-axis, I'd like to be able to define as many as I wish (but no more than there are entities, of course).

A concrete example would be a card with my Speedtest results. The download axis would range from 600 to 1000, the upload axis from 40 to 60 (only, I know) and the ping from 10 to 30.

The idea is that I don't really care about the values but about the shape of the line. If I display the upload and the ping on the same axis (which doesn't make very much mathematical sense anyway as it's not the same unit), I'll loose "definition" on the ping. And if I put the ping on its own and the two speed on the same axis, the upload speed will always be a line near the bottom and won't show any "definition".

My idea would be 100% retro-compatible with current yaml because you'd define the name of the axis with the  `y_axis` parameter. So `y_axis: secondary` would define a second axis called "secondary" but you could also define `y_axis: ping` and ` y_axis: upload` to have their name match their use case. Hopefully, that means `lower_bound_ping` or `upper_bound_upload`

**Comment by ildar170975** [comment]
The problem is that we have to live in 3D space and thus currently using 2D presentation for graphs.
It is rather difficult to have three Y-axis with different bounds - maximum 2 as it is implemented now.
Alternatively - someone might want to draw a third Y axis:
![image](https://github.com/user-attachments/assets/6562fe3d-5258-4209-a2d0-5a5cddc945f8)
which imho is ugly.

**Comment by Nickduino** [comment]
Oh yeah, my idea works a lot better if you **don't** show the labels.
Again, the idea is to see the shape of a line to check for a dip in speed for instance: you don't really care about the values

**Comment by ildar170975** [comment]
Your proposal makes the card inconsistent: smth works OK (for you) only if other options are not set.
So, this particular proposal cannot be accepted.

**Comment by Nickduino** [comment]
That's not true, you can still display the labels if you wish, it's just that it doesn't make much sense in this scenario.

**Comment by ildar170975** [comment]
Displayed labels should always make sense.

**Comment by ildar170975** [comment]
Closing until a better proposal is added

**Comment by Nickduino** [comment]
Do you get a bonus for the number of issues you close (and **don't** solve)? It's incredible how quick you are to reject others
